telegram/ext/dispatcher.py

```diff
@@ -227,8 +226,8 @@ def stop(self):
                 sleep(0.1)
             self.__stop_event.clear()
 
-        # async threads must be join()ed only after the dispatcher thread was joined,
-        # otherwise we can still have new async threads dispatched
+# async threads must be join()ed only after the dispatcher thread was joined,
+# otherwise we can still have new async threads dispatched
```

Comment:
That's weird, but our pre-commit hooks have given us lots of trouble already...
Could you fix the alignment and skip the pre-commit hooks by doing `git commit --no-verify`?
Everything else looks good, will be ready to merge after this.